llvm/utils/TableGen/DecoderEmitter.cpp

```diff
@@ -2561,7 +2561,7 @@ namespace llvm {
   // Emit the main entry point for the decoder, decodeInstruction().
   emitDecodeInstruction(OS, IsVarLenInst);
 
-  OS << "\n} // end namespace llvm\n";
+  OS << "\n} // end anonymous namespace\n";
```

Comment:
(nit) I believe this should read `// namespace` (this is what `clang-format` would insert). `// end [[of ]name ] namespace` syntax seems to be accepted for compatibility reasons.

Reply:
Done.